apiary.apib

```diff
@@ -397,6 +397,10 @@ We consider the following changes to be backwards-incompatible:
 We list all backwards-compatible additions here. These are currently available in all published versions.
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
+### February 2022
```

Comment:
PP: We've been using Latest for the current month.

Reply:
We stopped doing that recently, right? 🤔  To avoid we need to update it every month?